plugins/source/aws/resources/services/lambda/functions.go

```diff
@@ -153,9 +152,7 @@ func resolveCodeSigningConfig(ctx context.Context, meta schema.ClientMeta, resou
 	svc := cl.Services(client.AWSServiceLambda).Lambda
 
 	// skip getting CodeSigningConfig since containerized lambda functions does not support this feature
-	// value can be nil if the caller doesn't have GetFunctionConfiguration permission and only has List*
```

Comment:
This is still the case but theres a nil check for `r.Configuration` on https://github.com/cloudquery/cloudquery/pull/14729/files#diff-93c24c91e5c0e22f86d6415f1b98b47062f7d8974b9337c0b13c01322b8df981L149